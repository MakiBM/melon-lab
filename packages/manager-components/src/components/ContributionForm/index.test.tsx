import React from 'react';
import ContributionForm from './container';

const initialProps = {
  initialValues: {
    amount: '',
    total: '',
  },
  dataValid: true,
  melonAssetSymbol: 'MLN',
};

describe('ContributionForm', () => {
  const defaultElement = <ContributionForm {...initialProps} />;

  it('should render correctly', () => {
    const wrapper = shallow(defaultElement);
    expect(wrapper).toMatchSnapshot();
  });
});
