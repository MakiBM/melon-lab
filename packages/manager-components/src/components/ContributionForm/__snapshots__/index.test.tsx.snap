// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContributionForm should render correctly 1`] = `
<Formik
  dataValid={true}
  enableReinitialize={true}
  handleSubmit={[Function]}
  initialValues={
    Object {
      "amount": "",
      "total": "",
    }
  }
  isInitialValid={false}
  melonAssetSymbol="MLN"
  onSubmit={[Function]}
  render={[Function]}
  validateOnBlur={true}
  validateOnChange={true}
  validationSchema={[Function]}
/>
`;
